docs/070-project-management/leave-requests-and-stepping-away.md

```diff
@@ -0,0 +1,35 @@
+# Leave Requests and Stepping Away: Guidelines for Project Managers
+
+Guidelines that Project Managers may use to coach teams on communications for OOO and stepping away:
+
+## Out Of Office (OOO) Process
```

Comment:
@hhartingrex @akaroleff @farooqzakhilwal @dmundra this change seems to have removed anything about project managers pre-approving requests for time off, which in our current process happens before the formal request is placed in Unanet.
This seems like a critical step in the process to me, as project managers are the primary people who would be aware of the impacts requested time off would have on the project in terms of contract deliverables, SLAs, release schedules etc. I think we need to make it clearer that project managers are responsible for managing this, and proactively working with their team to alter time off schedules as needed (which might mean denying requests at times) to ensure projects can continue to meet their obligations.

Reply:
Agreed and thank you @grugnog . Change inserted, hopefully correctly.

Reply:
I think probably more detail we could add here, but that covers it for now I think - thanks!